### PR TITLE
Initialized Django application, added contributors to README.md

### PR DESCRIPTION
**Pull request by BrianRuizy**

### CODE REVIEW

**Comment by BrianRuizy** [comment]
image

**Review by brianraila — APPROVED** [review]
Looks good.

**Comment by soham1sawant** [comment]
Nice !!!

File: BookOmatic/settings.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '9d4mrj=v69x=lm=rvp8bmo)0dgt)fxp#j_8-w$t=2jb_h(&48%'
```
Comment (mmphego) [inline BookOmatic/settings.py:23]: Perhaps use pyenv to handle secrets.

File: BookOmatic/settings.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+]
+
```
Comment (mmphego) [inline BookOmatic/settings.py:41]: Too many spaces/

**Review by mmphego — CHANGES_REQUESTED** [review]

**Comment by KalleHallden** [comment]
This was really great, good work!

**Pull request merged**